### DISCUSSION TITLE
Naruto storm 3 & 4 please

### DISCUSSION THREAD

**Discussion by Endra999111** [skyline-emu/skyline#1661] [Game Requests]
Black screen

**Comment by ghost** [comment]
In the menus at least. Follow the game list for updates.

**Comment by javiabde24** [comment]
tengo un galaxy a21s podre correr skyline alli

**Reply by JERD001** [reply]
Es muy poco potente para skyline, no vas a mover mucho por no decir que no vas a mover nada al 100%.

**Comment by jin81790** [comment]
DLC/Update support and also pirate warriors 4 please
